regenerate.sh

```diff
@@ -24,29 +24,95 @@ trap finish EXIT
 
 export GOBIN=${WORKDIR}/bin
 export PATH=${GOBIN}:${PATH}
-mkdir -p ${GOBIN}
+mkdir -p "${GOBIN}"
 
 echo "remove existing generated files"
 # grpc_testing_not_regenerate/*.pb.go is not re-generated,
 # see grpc_testing_not_regenerate/README.md for details.
-rm -f $(find . -name '*.pb.go' | grep -v 'grpc_testing_not_regenerate')
+find . -name '*.pb.go' | grep -v 'grpc_testing_not_regenerate' | xargs -I{} sh -c 'rm -f {} || true'
```

Comment:
why do we need this? 

Also I think https://github.com/grpc/grpc-go/pull/6910 when merged will get rid of 'grpc_testing_not_regenerate' since Doug had a previous change that renames all of them to `pb.go`